src/Helpers/IsLoggedin.js

```diff
@@ -0,0 +1,8 @@
+const IsLoggedIn = () => {
```

Comment:
What do you think of having the `isLoggedIn` && the `logout` helper functions in 1 file and maybe call it `authhelpers.js` since they are both related to authentication functionality

Reply:
Consolidated into `authHelpers.js